modules/test/playwright/tests/object-web/rootModels.spec.ts

```diff
@@ -541,3 +541,214 @@ test.describe('Manage root models elements through Objects Admin', () => {
 		}
 	});
 });
+
+test.describe('Manage root models elements through Model Builder', () => {
+	test('assert inherited relationship styles on nodes and edges', async ({
```

Comment:
from the two tests you've created here, couldn't it be only one? because to test the styles you need to create the objects and you could do this already in the modal, testing both things; what are your thoughts behind this?

Reply:
I initially did it on the same test, but I thought that it would be better to separate it because each test is related to different stories, the styles are related to this one and the use of the modal on model builder is related to the [story](https://liferay.atlassian.net/browse/LPD-36875) that was discarded because the first one Carlos developed already covered it, so he asked me to include this test here. That doesn't prevent us from doing the asserts on the same test, but it made more sense to me to keep it separate considering the stories use cases. Let me know what you think and if necessary I'll change this!

Reply:
i understand your point of view! at the same time, even if they are from different stories it doesn't necessary needs to be two different tests; if they cover partially the same steps to achieve something, they could be the same test!

i have some considerations:

- the clean up from the other test, shouldn't it be inside an `afterEach` or something? because if the tests fails in the middle, it won't do the clean up right?
- this test related to the style of the card couldn't be a unit test? because it feels too much effort in one test just to see the styling and classes

i still think that this should be one test only, but considering the effort you've already put and the urgency of this story, i suggest you to send like this and after you can do a follow-up PR to change it; we can discuss this with the other fronts as well. but for now, you have my ok to forward it!

Reply:
Regarding your considerations:

1. That's a good question, I'm not sure about the answer, this was the pattern that we initially followed on this file and I don't know if it can be done in the way you mentioned because I'm unaware of the clean up logic which is on the headless side now, they removed our after each logic's and now we just pass the relationship to the api helper in each test. I'm not sure if it would work, but I can give it a try!
2. We tried doing an unit test, but it was taking too much effort to render the node, for every error that we fixed there was a new one, since we don't have any unit tests for it yet. Considering the story deadline we decided to move on with an playwright one.

Reply:
i see!! we can discuss on the reunion like you said :blush:

Reply:
- I've confirmed that the clean up still occurs even when the test fails, so we're secure with that!
- The style test will remain an functional one, but I will work on an follow up pr as you mentioned, to unite the tests.
Thank you for your amazing review, Beca!! 🌷 💟

Reply:
thanks too lari! :purple_heart: amazing PR! :cherry_blossom: